firebase-firestore/src/androidTest/assets/conformance/README.md

```diff
@@ -0,0 +1,9 @@
+# Conformance Tests
+
```

Comment:
optional: include directions on how to run the tests. Do they run as part of running the unit or integration tests, or is it a separate command?

Reply:
I will add a comment once these tests can indeed be run.